va/va.h

```diff
@@ -781,6 +781,11 @@ typedef enum
      * implementation, multiple frames encode/decode can improve HW concurrency
      */
     VAConfigAttribMultipleFrame         = 40,
+    /** \brief priority setting for the context. Read-Write
+     *  attribute value is \c VAConfigAttribValContextPriority
+     *  when read the value, it will return the range of priorty values
+     */
+    VAConfigAttribContextPriority       = 41,
```

Comment:
Is there any value for VAConfigAttribContextPriority? High? Medium and Low? For example, VAConfigAttribRateControl has the below values: VA_RC_NONE, VA_RC_CBR, etc.

Reply:
@FurongZhang , KMD could set from -1023 to 1023. so it is range, driver will report the range from 0, maybe 0 to 2046, , application could set any value from 0 to 2046, then it is converted to -1023 to 1023.  maybe for other vendor , it will be different, we are considering whether 3 or 5 level is enough.

Reply:
Out of curiosity, why we don't align with KMD -1023 to 1023?

Reply:
it depend on driver implementation, driver could report any range from 0